spec/source_file_branch_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require "helper"
+
+if SimpleCov.usable?
+  describe SimpleCov::SourceFile::Branch do
+    let(:results) do
```

Comment:
this `results` confuses me a bit, I think it's only used in the other `let`s to do `result[index]` - coudln't we just inline them then? Maybe I just don't understand its purpose :sweat_smile:

Reply:
I will modify it.